src/node_process_methods.cc

```diff
@@ -163,9 +163,18 @@ static void Cwd(const FunctionCallbackInfo<Value>& args) {
   size_t cwd_len = sizeof(buf);
   int err = uv_cwd(buf, &cwd_len);
   if (err) {
-    return env->ThrowUVException(err, "uv_cwd");
+    std::string err_msg =
+        std::string("process.cwd failed with error ") + uv_strerror(err);
+    if (err == UV_ENOENT) {
+      // If err == UV_ENOENT it is necessary to notice the user
+      // that the current working dir was likely removed.
+      err_msg =
+          err_msg +
+          std::string(", the current working directory was likely removed ") +
+          std::string("without changing the working directory");
```

Comment:
Completely optional, but this could be simplified a bit:

```suggestion
          ", the current working directory was likely removed "
          "without changing the working directory";
```